iceberg/iceberg-handler/src/test/queries/positive/iceberg_major_compaction_single_partition.q

```diff
@@ -29,19 +29,19 @@ create table ice_orc_wo_evo (
     first_name string, 
     last_name string
  )
-partitioned by (dept_id bigint, 
+partitioned by (dept_id decimal(10, 2),
```

Comment:
dept_id can't be a decimal, why change here?

Reply:
Same, to increase test coverage of partition field data types in iceberg compaction. Yes, `dept_id` wouldn't likely be a decimal in a real world, but it is only a field name which doesn't affect anything?

Reply:
closing as duplicate issue